support/patchTreeShaking.ts

```diff
@@ -0,0 +1,22 @@
+(async function () {
+  const {
+    promises: { readFile, writeFile }
+  } = await import("fs");
+  const { dirname, normalize } = await import("path");
+  const { quote } = await import("shell-quote");
+  const { fileURLToPath } = await import("url");
+
+  // patch needed due to Stencil tree-shaking bug
```

Comment:
Nitpick: maybe move this to the top of the function since it is describing the whole script. Up to you.